src/index.js

```diff
@@ -44,12 +44,34 @@ function bodyIsJson(httpResponse) {
   return thisContentType && thisContentType.includes(jsonContentType);
 }
 
+// Allows a user to use "orgId" in the custom header block rather than 'x-gw-ims-org-id'
+function formatOrgIdHeader(customHeaders) {
+  const finalHeaders = {
+    ...customHeaders,
+    'x-gw-ims-org-id': customHeaders.orgId
+  };
+  delete finalHeaders.orgId;
+
+  return finalHeaders;
+}
+
 export default class Reactor {
   constructor(accessToken, userOptions = {}) {
-    const options = Object.assign({}, defaultReactorOptions, userOptions);
+    const options = {
+      ...defaultReactorOptions,
+      ...userOptions
+    };
     this.baseUrl = removeTrailingSlash(options.reactorUrl);
     this.enableLogging = options.enableLogging;
-    this.headers = this.reactorHeaders(accessToken);
+    this.customHeaders = options.customHeaders || {};
```

Comment:
let's have the user supply the real header name (update the README as well) so there's no confusion on which headers should be supplied with the real name VS some kind of abstracted "nicer" name.